action.yml

```diff
@@ -48,7 +48,13 @@ inputs:
     description: '[Optional] The directory to wipe and replace in the target repository'
     default: ''
     required: false
-        
+  create-target-branch-if-needed:
+    type: boolean
```

Comment:
Cast as a `boolean` for clarity but passed as a `string` in the dockerfile and entrypoint script.